cmd/local-gadget/profile/block-io.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package profile
+
+import (
+	"github.com/cilium/ebpf"
+	"github.com/spf13/cobra"
+
+	commonprofile "github.com/inspektor-gadget/inspektor-gadget/cmd/common/profile"
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/utils"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-collection/gadgets/profile"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/block-io/tracer"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/block-io/types"
+)
+
+func newBlockIOCmd() *cobra.Command {
+	var commonFlags utils.CommonFlags
+
+	runCmd := func(*cobra.Command, []string) error {
+		blockIOGadget := &ProfileGadget[types.Report]{
```

Comment:
Since this gadget doesn't support filtering, shouldn't we check if any filter is set? (this is done on the server side for the kubectl-gadget case https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadget-collection/gadgets/profile/block-io/gadget.go#L89)

Reply:
Like `--containername` and `--runtimes` ?

Reply:
Right, and then `--namespace`, `--podname` in the future.